custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -243,9 +255,12 @@ def get_payload(self, repeat_record, test_case):
 
         test_case_properties = test_case.dynamic_case_properties()
         episode_case_properties = episode_case.dynamic_case_properties()
+        use_new_2b_app_structure = self.use_new_2b_app_structure(person_case, episode_case)
 
         interval_id, lab_serial_number, result_grade, dmc_code = self._get_mandatory_fields(
-            test_case, test_case_properties)
+            test_case, test_case_properties, occurence_case,
+            use_new_2b_app_structure
```

Comment:
since you made this an instance method, you could avoid passing this variable around and just call `self.use_new_2b_app_structure`